tests/unit-tests/test-class-sensei-course-progression-settings.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+class Test_Sensei_Course_Progression_Settings extends WP_UnitTestCase {
+	public function testMetaFieldsExist() {
+		$settings = Sensei_Course_Progression_Settings::instance();
+		$settings->register_post_meta();
+		$keys = get_registered_meta_keys( 'post', 'course' );
+		$this->assertArrayHasKey( '_video_course_autocomplete', $keys );
```

Comment:
When there are multiple assertions in the same test case, I like to add a short message as the third parameter so that it's easier to identify which one is failing.